src/compat/types_type_kind_ge_000.ml

```diff
@@ -0,0 +1,3 @@
+type type_kind_407 = Types.type_kind
```

Comment:
Does that `407` version make sense? I took heavy inspiration from what currently exists but I'm not sure about what that version should represent

Reply:
Perhaps the version before it started to diverge? So in that case maybe `412` should be used instead?

Reply:
Yes, `412` would make more sense here.